amethyst_renderer/src/mesh.rs

```diff
@@ -150,3 +152,7 @@ where
         })
     }
 }
+
+impl Component for Mesh {
+    type Storage = VecStorage<Self>;
```

Comment:
Any reason for Vec over DenseVec?

Reply:
It's justbthe same `MeshComponent` uses now.
I haven't dive deep into "what `Storage` to use?" yet.